client/src/pages/OnboardingPage/OnboardingPage.css

```diff
@@ -0,0 +1,75 @@
+.OnboardingPage {
+  font-family: var(--default-font);
+  background: linear-gradient(180deg, rgba(255, 255, 255, 0) 72.4%, rgba(255, 255, 255, 0.75) 100%),
+    #ffcb08;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+}
+.OnboardingPage__wrapper {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+.OnboardingPage .Logo {
+  width: 100%;
+  height: 16.5rem;
+  background-size: contain;
+}
```

Comment:
If you need to modify another component, then update the code for that component. We don't want to override classes inside other components, because then our CSS will become impossible to maintain